src/components/Section/Foreign/Activities/BenefitActivity/FutureBenefit.jsx

```diff
@@ -43,92 +41,99 @@ export default class FutureBenefit extends ValidationElement {
       Reason: this.props.Reason,
       Obligated: this.props.Obligated,
       ObligatedExplanation: this.props.ObligatedExplanation,
-      ...queue
+      ...queue,
     })
   }
 
-  updateBegin(values) {
+  updateBegan(values) {
     this.update({
-      Begin: values
+      Began: values,
     })
   }
 
   updateFrequency(values) {
     this.update({
-      Frequency: values
+      Frequency: values,
     })
   }
 
   updateOtherFrequency(values) {
     this.update({
-      OtherFrequency: values
+      OtherFrequency: values,
     })
   }
 
   updateCountry(values) {
     this.update({
-      Country: values
+      Country: values,
     })
   }
 
   updateValue(values) {
     this.update({
-      Value: values
+      Value: values,
     })
   }
 
   updateValueEstimated(values) {
     this.update({
-      ValueEstimated: values
+      ValueEstimated: values,
     })
   }
 
   updateReason(values) {
     this.update({
-      Reason: values
+      Reason: values,
     })
   }
 
   updateObligated(values) {
     this.update({
-      Obligated: values
+      Obligated: values,
     })
   }
 
   updateObligatedExplanation(values) {
     this.update({
-      ObligatedExplanation: values
+      ObligatedExplanation: values,
     })
   }
 
   render() {
+    // TODO
+    // One of the things to go back and fix is the grammar.
+    // The backend is expecting Began, which is past tense
+    // Future should be Begin
```

Comment:
The names of an object keys shouldn't change based on the values in that object. IMO this can all be avoided by not worrying about tense, use naming like `Start, End`.